src/genomics-england/genomics-england.py

```diff
@@ -141,7 +141,7 @@ def main():
         short_id = re.sub(
             r'_+', '_', category.lower().replace(' ', '_').replace('/', '_').replace(';', '').replace('(', '').replace(
```

Comment:
Can we stop replacing underscores?

Reply:
This replaces spaces and other weird characters with underscores for the broad categories, that do not have IDs. Most of them do, but there are a couple of categories that I had to create IDs for.

Reply:
Ok